substrate/validator-sets/pallet/src/lib.rs

```diff
@@ -835,6 +879,18 @@ pub mod pallet {
     }
   }
 
+  impl<T: Config> AllowMint for Pallet<T> {
+    fn allow(balance: &Balance) -> bool {
```

Comment:
is_allowed instead of allow? I'd expect allow to take in a struct's &mut self and whitelist something, not check if it's allowed.